subsystem/kubeapi_test.go

```diff
@@ -855,6 +870,84 @@ var _ = Describe("[kube-api]cluster installation", func() {
 		}, "2m", "2s").Should(Equal(0))
 	})
 
+	FIt("deploy CD with ACI and agents with ignitionEndpoint - wait for ready, delete CD and verify ACI and agents deletion", func() {
+		deployClusterDeploymentCRD(ctx, kubeClient, clusterDeploymentSpec)
+		deployInfraEnvCRD(ctx, kubeClient, infraNsName.Name, infraEnvSpec)
+		caCertificateSecretName := "ca-certificate"
+		deploySecret(ctx, kubeClient, caCertificateSecretName, map[string]string{corev1.TLSCertKey: "abc"})
+		modifiedAciSpec := aciSpec.DeepCopy()
+		modifiedAciSpec.IgnitionEndpoint = &hiveext.IgnitionEndpoint{
+			Url: "example.com",
```

Comment:
`            "message": "The Spec could not be synced due to an input error: The URL scheme must be http(s) and specified in the URL: 'example.com'"`

Reply:
Where did you see this error?

Reply:
In the CI result, click `open stdout`

blob:https://prow.ci.openshift.org/6a9468b0-7d81-456f-90af-d2b0141f493c
![Screenshot from 2021-12-23 12-09-00](https://user-images.githubusercontent.com/12015519/147224982-a6aa3a8a-c794-46a2-a358-b552a5e11746.png)